doc/specifications/rest/REST-API-V2.rst

```diff
@@ -2062,6 +2062,10 @@ or if destination is /content/trash
         HTTP/1.1 201 Created
         Location: /content/trash/<ID>
 
+.. code:: http
```

Comment:
Shouldn't this explain the two cases? (201 if trashing single location content, 204 otherwise.)